### ISSUE TITLE
API Key now?

### ISSUE THREAD

**Issue by elizabethsiegle** [caneroj1/clarifai-hs#3]

**Comment by maxcell** [comment]
Yeah you are right @elizabethsiegle! [In the last month, Clarifai has moved over to API Keys](http://blog.clarifai.com/introducing-api-keys-a-safer-way-to-authenticate-your-applications/#.WW4didPyvUI)! It would be great to see this implemented into the client, even though the client is really only focused around our v1 API. Definitely feel free to open a PR for this and make the contributions to keeping this up-to-date 😄 .

**Comment by caneroj1** [comment]
Okay, so it looks like we'll have to support the API key change! I'm open to receiving pull requests for this change, or when I have some time I'll make the change myself.

I'd also like to take another look at this library and see what else needs to be updated, such as additional API requests and functionality.
